chainer/link.py

```diff
@@ -310,16 +310,16 @@ def to_cpu(self):
         self._device_id = None
         return self
 
-    def to_gpu(self, device=None):
+    def to_gpu(self, device_id=None):
```

Comment:
In the original implementation, `to_gpu` method accept `Device` object as an argument. We can't change this behavior.